src/storage/StorageConfig.ts

```diff
@@ -44,20 +44,19 @@ export class StorageConfig {
     listeners: StorageConfigListener[]
     nodeId: string
     apiUrl: string
-    _poller?: NodeJS.Timeout
-    _stopPoller: boolean
+    private _poller!: ReturnType<typeof setTimeout>
```

Comment:
Adding the `!` here removes need to add it everywhere else.
`ReturnType<typeof setTimeout>` is basically same thing as `NodeJS.Timeout` except it's not coupled directly to Node.